### DISCUSSION TITLE
Jekyll and github pages - not building but no error either

### DISCUSSION THREAD

**Discussion by hudyu17** [community/community#23798] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, I’m trying to host a Jekyll site on Github Pages but it doesn’t appear to be building (screenshot) </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/446fbd14880fc223b79f13266fae393ecff1f4c6.png" data-download-href="/uploads/short-url/9LpRK4ITirF9gkTft6Lze5tv9e6.png?dl=1" title="gh pages" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/48703806/181079364-d7b5fd28-c006-4566-9715-ac22aebe24d3.png" alt="gh pages" data-base62-sha1="9LpRK4ITirF9gkTft6Lze5tv9e6" width="690" height="408" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/446fbd14880fc223b79f13266fae393ecff1f4c6_2_690x408.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/446fbd14880fc223b79f13266fae393ecff1f4c6_2_1035x612.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/446fbd14880fc223b79f13266fae393ecff1f4c6.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/446fbd14880fc223b79f13266fae393ecff1f4c6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gh pages</span><span class="informations">1257×745 47.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> Nothing’s happened after 12 hours but I’m not getting any build errors either.
<p>It’s building on the gh-pages branch, as it should, so I’m thinking it might have something to do with my gemfile. When hosting locally, I have to run <code>bundle exec jekyll serve</code> as some gems need to be downgraded e.g. i18n - I’ve read that github pages runs bundle --safe and doesn’t actually read my gemfile.lock, so I’m not sure if this is causing the issue.</p>
<p>Thanks!</p>

**Comment by hudyu17** [comment, marked as answer]
<p>SOLVED: turns out you need to select a random jekyll theme, even if you’re using a different theme.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for coming back to share what worked <a class="mention" href="/hudyu17">@hudyu17</a>, and welcome to the community!</p>
